client/java/.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
```

Comment:
This has to be integrated into the repo root directory, a good name would be `.github/workflow/java-client.yml`.

Reply:
updated